Formula/godep.rb

```diff
@@ -23,16 +23,20 @@ def install
   end
 
   test do
-    ENV["GO15VENDOREXPERIMENT"] = "0"
     ENV["GOPATH"] = testpath.realpath
-    (testpath/"Godeps.json").write <<-EOS.undent
+    (testpath/"Godeps/Godeps.json").write <<-EOS.undent
       {
         "ImportPath": "github.com/tools/godep",
-        "GoVersion": "go1.6",
-        "Deps": []
+        "GoVersion": "go1.7",
+        "Deps": [
+          {
+            "ImportPath": "go.googlesource.com/tools",
+            "Rev": "3fe2afc9e626f32e91aff6eddb78b14743446865"
+          }
+        ]
       }
     EOS
-    (testpath/"src/foo/bar/Godeps").install "Godeps.json"
-    cd("src/foo/bar") { system bin/"godep", "path" }
+    system bin/"godep", "restore"
+    assert File.exist?("src/go.googlesource.com/tools/README")
```

Comment:
Trying to get this done fairly quickly; also working on GPG stuff in the background which is more important. Just wanted to push this through ASAP so I didn't wake up to 450 thumbs up in the other thread later.